Conan.VisualStudio.Core/ConanRunner.cs

```diff
@@ -101,6 +101,7 @@ public ProcessStartInfo Install(ConanProject project, ConanConfiguration configu
                 RedirectStandardError = true,
                 CreateNoWindow = true
             };
+            startInfo.EnvironmentVariables.Remove("VisualStudioVersion");
```

Comment:
Need to double-check: this only affects the environment for this specific `ProcessStartInfo`, right?

Reply:
yes, exactly